app/src/pages/Lobby/WaitingRoom.tsx

```diff
@@ -27,7 +27,8 @@ function WaitingRoom() {
     canSeeStartGameButton &&
     currentGame.seconds_per_turn &&
     currentGame.num_entries_per_player &&
-    playersWithUsernames.length >= MIN_NUMBER_OF_PLAYERS
+    playersWithUsernames.length >= MIN_NUMBER_OF_PLAYERS &&
+    find(playersWithUsernames, player => player.id === currentPlayer.id)
```

Comment:
the host must have a user name to start! Otherwise they would be attempted to be booted out upon clicking "Everyone's Here"